### PR TITLE
[Doc-16490] Changed some notes about python gateway service

### LINKED ISSUE #16490: [Doc][GUIDE] Changed some notes about python gateway service
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Hi,
I found that the default value of python-gateway.enabled in the config file is false, which means it won't start with the server by default. So the description in the documentation is wrong.

[config of python-gateway](https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-api/src/main/resources/application.yaml#L145-L147)

And I found this note was missing from the Chinese documentation.

### Documentation Links

[en doc](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/installation/standalone.md?plain=1#L37-L52)
[zh doc](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/zh/guide/installation/standalone.md?plain=1)

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by BruceWong96**
## Purpose of the pull request

Fix doc for standalone.
close #16490 

## Verify this pull request

This pull request is code cleanup without any test coverage.

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: docs/docs/zh/guide/installation/standalone.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ bash ./bin/dolphinscheduler-daemon.sh stop standalone-server
 bash ./bin/dolphinscheduler-daemon.sh status standalone-server
 ```
 
+> **_注意_**: Python 网关服务默认不会随 api-server 一起启动，
```
Comment (SbloodyS) [inline docs/docs/zh/guide/installation/standalone.md]:
```suggestion
> **_注意_**: Python 网关服务默为关闭状态
```

File: docs/docs/en/guide/installation/standalone.md [hunk 2]
```diff
@@ -47,8 +47,8 @@ bash ./bin/dolphinscheduler-daemon.sh stop standalone-server
 bash ./bin/dolphinscheduler-daemon.sh status standalone-server
 ```
 
-> Note: Python gateway service is started along with the api-server, and if you do not want to start Python gateway
-> service please disabled it by changing the yaml config `python-gateway.enabled : false` in api-server's configuration
+> Note: Python gateway service is not started along with the api-server by default. If you want to start the Python gateway
```
Comment (SbloodyS) [inline docs/docs/en/guide/installation/standalone.md]:
```suggestion
> Note: Python gateway service is disabled by default. If you want to start the Python gateway
```

**Review by SbloodyS — APPROVED** [review]
LGTM

**Review by Gallardot — APPROVED** [review]
LGTM

**Review by ruanwenjun — APPROVED** [review]
+1, good job.

**Pull request merged**